__tests__/server.test.js

```diff
@@ -0,0 +1,47 @@
+const format = require("pg-format")
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed.js')
+const request = require('supertest')
+const express= require('express');
+const app = require("../app/app.js");
+const jestSorted = require('jest-sorted');
+
+const { articleData, commentData, topicData, userData } = require('../db/data/test-data/index')
+
+
+beforeAll(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+
+describe("Testing APIs", () => {
+    test("Status  200 and return all topics", () => {
+      return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({body}) =>{
+        expect(body).toHaveLength(3)
+
+
```

Comment:
This can be 1 test. You don't need to test the status and the data separately when integration testing. This is because, you will have to go through the whole MVC flow to pass this test. Then your test below will pass straight away!